pkg/broker/ingress/ingress_handler.go

```diff
@@ -182,6 +189,27 @@ func (h *Handler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
 
 	ctx := request.Context()
 
+	// jwt, err := h.oidcTokenHandler.GetJWT(types.NamespacedName{
+	// 	Name:      "mt-broker-ingress",
+	// 	Namespace: "knative-eventing",
+	// }, "my-audience")
+	// if err != nil {
+	// 	h.Logger.Error("could not get token", zap.Error(err))
+	// 	writer.WriteHeader(http.StatusInternalServerError)
+	// 	return
+	// }
+
+	// h.Logger.Info("got a JWT", zap.String("jwt", jwt))
+
+	// idToken, err := h.oidcTokenHandler.VerifyJWT(ctx, jwt, "my-audience")
+	// if err != nil {
+	// 	h.Logger.Error("could not verify token", zap.Error(err))
+	// 	writer.WriteHeader(http.StatusInternalServerError)
+	// 	return
+	// }
+
+	// h.Logger.Info("verified token", zap.Any("idtoken", idToken))
```

Comment:
Similar question here, do we want to leave this commented out code in?

Reply:
Same as above. Was for testing and got accidentally committed :facepalm: